src/NzbDrone.Core/Parser/IsoLanguages.cs

```diff
@@ -41,7 +41,10 @@ public static class IsoLanguages
                                                                new IsoLanguage("bg", "", "bul", Language.Bulgarian),
                                                                new IsoLanguage("ml", "", "mal", Language.Malayalam),
                                                                new IsoLanguage("uk", "", "ukr", Language.Ukrainian),
-                                                               new IsoLanguage("sk", "", "slk", Language.Slovak)
+                                                               new IsoLanguage("sk", "", "slk", Language.Slovak),
+                                                               new IsoLanguage("th", "th", "tha", Language.Thai),
+                                                               new IsoLanguage("pt", "br", "por", Language.PortugueseBrazil),
+                                                               new IsoLanguage("es", "mx", "spa", Language.SpanishLatino)
                                                            };
 
         public static IsoLanguage Find(string isoCode)
```

Comment:
Find function needs a bit of tweaks to fully support this. Currently it selects the first Language based on the two letter code without looking at the country code. 

Radarr version tweaks this a bit to breakdown the incoming string and parse accordingly if it gets a "pt-BR" formatted `isoCode`